allensdk/brain_observatory/stimulus_analysis.py

```diff
@@ -581,3 +581,13 @@ def row_from_cell_id(self, csid=None, idx=None):
         else:
             raise Exception("Could not find row for csid(%s) idx(%s)" % (str(csid), str(idx)))
     
+    
+def nonraising_ks_2samp(data1, data2, **kwargs):
+    """ scipy.stats.ks_2samp now raises a ValueError if one of the input arrays 
```

Comment:
It might be more helpful if you referred to a particular version of scipy when change happened, for future reference.